src/tools/curve_cli.cpp

```diff
@@ -35,6 +35,8 @@ DEFINE_string(peer,
               "", "Id of the operating peer");
 DEFINE_string(new_conf,
               "", "new conf to reset peer");
+DEFINE_bool(remove_copyset, false, "Whether need to remove broken copyset "
```

Comment:
Actually, it's a dangerous action to delete copyset even if it's a broken copyset, we want to let user know what he is doning now, instead of silently deleting it.

Reply:
> why default is false? if the broken copyset data keep there, there will some problem

Actually, it's a dangerous action to delete copyset even if it's a broken copyset, we want to let user know what he is doning now, instead of silently deleting it.